### ISSUE TITLE
[BUG]  Firefox crashes with "tab crash" after upgrade to Ubuntu 22.04

### ISSUE THREAD

**Issue by Randall-Coding** [microsoft/playwright#14972]
**Context:**
- Playwright Version: [Driver 1.12]
- Operating System: [e.g. Windows, Linux or Mac]
- Node.js version: [12.20.1]
- Browser: [Firefox] The browser downloaded is `firefox-1271/`
- Extra: [using playwright-rust 0.19 which uses your playwright driver 1.12]

**Code Snippet**
```rust
    let playwright = Playwright::initialize().await.unwrap();
    playwright.prepare().unwrap();
    let firefox = playwright.firefox();
    let browser: Browser = firefox.launcher().headless(false).launch().await.unwrap();
    let context = browser.context_builder().build().await.unwrap();
    let page = context.new_page().await.unwrap();  //crashes here
```

**Describe the bug**
Since upgrading to Ubuntu 22.04, running firefox in playwright immediately crashes my app with a tab error (see image below) when I first create a new page, which yields an error in console:

> ErrorMessage { name: "Error", message: "Page closed", stack: "Error: Page closed\n    at FFSession 

I know the playwright-rust bindings I'm using have an outdated version of playwright (it's the best there is AFAIK for Rust), but whether this is an ongoing issue or not, you seem like the best people to point me in the right direction. 

I'm not sure if this is an issue with firefox, with the playwright driver, or with Ubuntu itself and I'm not sure how to troubleshoot this.  

Although I have no problem using any browser I have installed normally, I only have a problem using the automated versions of first chrome and now firefox.

![firefox_tabcrash_ubuntu_22](https://user-images.githubusercontent.com/39175191/174520725-e7a940f8-0f0e-423f-b715-27d79acd10b1.png)

**Comment by Randall-Coding** [comment]
Update:  I found this issue which indicates Ubuntu 22.04 is not yet supported.  It is from January, but I suppose I should assume that is still the case?  https://github.com/microsoft/playwright/issues/11175#issuecomment-1005119734

**Comment by mxschmitt** [comment]
Ubuntu 22 is not supported yet, you are also using a very outdated Playwright version + a not official language binding. So not something where we can act from our side on :/

**Comment by Randall-Coding** [comment]
@mxschmitt  Thanks, and I understand playwright-rust was written by someone not affiliated with you guys.  However I still have more luck talking to you than the dev on that project who has been unresponsive for some time now.  

I'm thinking I will end up upgrading the Playwright version myself.  I already did upgrade somewhat by upgrading from 1.11 to 1.12 myself, but it appears after this point Playwright start command changed and no longer produces the stdout output that playwright-rust was relying on for certain information.  

Perhaps I can reach out again later to get some guidance on how to obtain that info? It would save me a lot of time as I know so little about the inner workings of the playwright driver.

**Comment by mxschmitt** [comment]
Most likely it's fixed if you upgrade to 1.24 which gets released soon.

Since it supports Ubuntu 22 officially: https://github.com/microsoft/playwright/issues/13738

Closing it for now until you can repro it after 1.24.
